src/learn/pages/Award.jsx

```diff
@@ -36,9 +36,16 @@ export default function Award() {
     history.push("");
   };
 
-  window.history.pushState(null, null, location.href);
-  window.addEventListener("popstate", (e) => {
-    history.go(1);
+  useEffect(() => {
+    const onUnload = (e) => {
+      e.preventDefault();
+      e.returnValue = "";
+    };
+    window.addEventListener("beforeunload", onUnload);
+    window.history.pushState(null, null, location.href);
```

Comment:
```suggestion
    window.history.pushState(null, null, window.location.href);
```

Linterが反応してるけどこれでエラーって起こるかな

Reply:
やってみます

Reply:
エラー起こりませんでした
修正しましたゞ